indy_node/server/config_req_handler.py

```diff
@@ -202,3 +206,20 @@ def get_auth_key(operation):
             AuthActionAdd(txn_type=auth_type,
                           field=field,
                           value=new_value).get_action_id()
+
+    @staticmethod
+    def get_auth_constraint(operation):
+        return ConstraintCreator.create_constraint(operation.get(CONSTRAINT))
+
+    def update_auth_constraint(self, auth_key, constraint):
+        self.state.set(auth_key.encode(),
+                       self.constraint_serializer.serialize(constraint))
+
+    def updateState(self, txns, isCommitted=False):
```

Comment:
I think we need to integrate it for the new Pluggable Request Handler for AUTH_RULE txn.